inst/extdata/config_schema.json

```diff
@@ -0,0 +1,191 @@
+                "seed",
+                "task_id"
+            ],
+            "title": "Epinow2"
+        },
+        "Data": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "blob_storage_container": {
+                    "type": ["null", "string"]
+                },
+                "report_date": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "date"
+                    }
+                },
+                "reference_date": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "date"
+                    }
+                }
+            },
+            "required": [
+                "blob_storage_container",
+                "path",
+                "reference_date",
+                "report_date"
+            ],
+            "title": "Data"
+        },
+        "Parameters": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "blob_storage_container": {
+                    "type": ["string", "null"]
+                }
+            },
+            "required": [
+                "blob_storage_container",
+                "path"
+            ],
+            "title": "Parameters"
+        },
+        "Priors": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "rt": {
+                    "$ref": "#/definitions/Rt"
+                },
+                "gp": {
+                    "$ref": "#/definitions/Gp"
+                }
+            },
+            "required": [
+                "gp",
+                "rt"
+            ],
+            "title": "Priors"
+        },
+        "Gp": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "alpha_sd": {
+                    "type": "number"
+                }
+            },
+            "required": [
+                "alpha_sd"
+            ],
+            "title": "Gp"
+        },
+        "Rt": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "mean": {
+                    "type": "integer"
+                },
+                "sd": {
+                    "type": "number"
+                }
+            },
+            "required": [
+                "mean",
+                "sd"
+            ],
+            "title": "Rt"
+        },
+        "SamplerOpts": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
```

Comment:
Can we add the number of samples run here as a parameter -- seems like something we could want to change at runtime, and I think helpful to encode it explicitly instead of using the defaults